plugins/fontsize/trumbowyg.fontsize.js

```diff
@@ -75,6 +75,38 @@
             });
             dropdown.push('fontsize_' + size);
         });
+        
+         var freeSizeButtonName = 'fontsize_custom',
+            freeSizeBtnDef = {
+                fn: function () {
+                    trumbowyg.openModalInsert('Custom Font Size',
+                        {
+                            size: {
+                                label: 'Font Size',
+                                value: '48px'
+                            }
+                        },
+                        function (values) {
+                            var text = trumbowyg.range.startContainer.parentElement
+                            var selectedText = trumbowyg.getRangeText();
+                            if($(text).html() == selectedText) {
+                                $(text).css('font-size', values.size);
+                            } else {
+                                trumbowyg.range.deleteContents();
+                                var html = '<span style="font-size: ' + values.size + ';">' + selectedText + '</span>';
+                                var node = $(html)[0];
+                                trumbowyg.range.insertNode(node);
+                            }
+                            trumbowyg.saveRange();
+                            return true;
+                        }
+                    );
+                },
+                text: '<span style="font-size: medium;">Custom</span>',
+                hasIcon: false
+            };
+        trumbowyg.addBtnDef(freeSizeButtonName, freeSizeBtnDef);
+        dropdown.push('fontsize_custom');
```

Comment:
Since its already assigned, i would use `freeSizeButtonName`:

```js
dropdown.push(freeSizeButtonName);
```

Reply:
I'll give this a shot